sdk/python/packages/flet/src/flet/cli/commands/run.py

```diff
@@ -74,9 +82,17 @@ def add_arguments(self, parser: argparse.ArgumentParser) -> None:
 
     def handle(self, options: argparse.Namespace) -> None:
         # print("RUN COMMAND", options)
-        script_path = options.script
-        if not os.path.isabs(options.script):
-            script_path = str(Path(os.getcwd()).joinpath(options.script).resolve())
+        if options.module:
+            script_path = str(options.script).replace(".", "/")
+            if os.path.isdir(script_path):
+                script_path += "/__main__.py"
```

Comment:
Would `/` work on Windows?

Reply:
Thankfully yes, I'm on windows and python can handle it apparently. Even when I mix `/` and `\` it seems.

Reply:
`os.path.join` ?